px-data-grid.html

```diff
@@ -1220,6 +1254,24 @@
 
           Polymer.RenderStatus.afterNextRender(this._vaadinGrid, () => this._vaadinGrid.notifyResize());
         }
+
+        /**
+         * Expand or collapse the details container for any item's row.
+         */
+        showRowDetails(item, isShowing) {
```

Comment:
should this be called setRowDetailsVisible, as it takes boolean and can be also used to hide